flink-cdc-connect/flink-cdc-source-connectors/flink-cdc-base/src/main/java/org/apache/flink/cdc/connectors/base/source/assigner/HybridSplitAssigner.java

```diff
@@ -137,6 +161,7 @@ public Optional<SourceSplitBase> getNext() {
                 // assigning the stream split. Otherwise, records emitted from stream split
                 // might be out-of-order in terms of same primary key with snapshot splits.
                 isStreamSplitAssigned = true;
+                enumeratorMetrics.enterStreamReading();
```

Comment:
If `isNewlyAddedAssigningFinished(snapshotSplitAssigner.getAssignerStatus())`, this should invoke `enumeratorMetrics.enterStreamReading();`.

Reply:
`enumeratorMetrics.exitStreamReading()` should be invoked in `addSplits`.

Reply:
> If `isNewlyAddedAssigningFinished(snapshotSplitAssigner.getAssignerStatus())`, this should invoke `enumeratorMetrics.enterStreamReading();`.

Yes, I did miss it. I have already made adjustments

Reply:
> `enumeratorMetrics.exitStreamReading()` should be invoked in `addSplits`.

Inside `snapshotSplitAssigner.addSplits`, it will handle

Reply:
I have already dealt with it